File: Algorithms/657.Robot-Return-to-Origin/robot_return_to_origin.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        res = [0, 0]
        for ele in moves:
            if ele == "U":
                res[1] += 1
            elif ele == "D":
                res[1] -= 1
            elif ele == "L":
                res[0] -= 1
            else:
                res[0] += 1
        return res[0] == 0 and res[1] == 0
                

File: Algorithms/657.Robot-Return-to-Origin/robot_return_to_origin.go
func judgeCircle(moves string) bool {
    x := 0
    y := 0
    for _, ele := range moves{
        switch string(ele) {
            case "U":
            y += 1
            case "D":
            y -= 1
            case "L":
            x -= 1
            case "R":
            x += 1
        }
    }
    if x == 0 && y == 0 {
        return true
    } else {
        return false
    }
}